cf-deployment.yml

```diff
@@ -454,15 +454,19 @@ instance_groups:
               key-1:
                 signingKey: "((uaa_jwt_signing_key.private_key))"
         clients:
+          cc_routing:
+            authorities: routing.router_groups.read
+            authorized-grant-types: client_credentials
+            secret: "((uaa_clients_cc-routing_secret))"
           cc-service-dashboards:
             authorities: clients.read,clients.write,clients.admin
             authorized-grant-types: client_credentials
             scope: openid,cloud_controller_service_permissions.read
             secret: "((uaa_clients_cc-service-dashboards_secret))"
-          cc_routing:
-            authorities: routing.router_groups.read
+          cc_service_key_client:
+            authorities: credhub.read,credhub.write
```

Comment:
I believe this line should be removed, as cf CredHub has ACLs enabled:
https://github.com/cloudfoundry/cf-deployment/blob/5ef5b808ad82c8251d9b093f6dce3df8b456b79f/operations/experimental/secure-service-credentials.yml#L37-L39
and authorities/scope sections of the UAA token is completely ignored by CredHub.

CredHub for BOSH uses those because it has ACLs disabled. You might have seen it in `bosh-deployment` manifests, but it's not applicable here and could be confusing to the operators.

I'm not entirely sure whether you should completely remove the line, or say something like `authorities: uaa.none`. It should probably be tested out first.

/cc @dsabeti @matt-royal @tcdowney

Reply:
Thanks @ishustava!

Reply:
Good to know, that was a misunderstanding on our part and we'll get our story updated. 

Thanks @ishustava!

Looks like these are the [relevant docs](https://github.com/cloudfoundry-incubator/credhub/blob/master/docs/authorization-phase1.md#supported-actors-aka-authenticated-identities) and as long as a credential in CredHub has `uaa-client: cc_service_key_client` we should be able to access it?

Reply:
@tcdowney 

We have tested it, and it looks like we still do require authorities. Sorry for the confusion!

/cc @dannysullivan

Reply:
Cool, thanks!